mobile2/src/components/AddBookForm.tsx

```diff
@@ -0,0 +1,88 @@
+import React, { Component } from "react";
+import { graphql, commitMutation } from "react-relay";
+import env from "../Environment";
+import { TextInput, Button } from "react-native-paper";
+import { View } from "react-native";
+import { styles } from "../constants";
+import { buildSchema } from "graphql";
+
+export default class AddBookForm extends Component<{}> {
+  state = {
+    bookName: "",
+    total: "",
+    read: "",
+  };
+
+  render() {
+    const mutation = graphql`
+      mutation AddBookFormMutation($input: CreateBookInput!) {
+        createBook(input: $input) {
+          name
+          numPages
+          currentPageNum
+        }
+      }
+    `;
+
+    const commit = (name, numPages, currentPageNum) => {
+      //Parameters of our mutation
+      const variables = {
+        input: {
+          name,
+          numPages,
+          currentPageNum,
+        },
+      };
```

Comment:
Can you move this to a separate Mutation file? There is a mutations folder which has all these files! 
Also, the Add Form here is the older one, I updated it to add the Status field too. You can pull the new changes and call this mutation there. 

Just a bit of moving around to do. All good otherwise :)

Reply:
Moved it to mutation folder